multicluster/controllers/multicluster/clusterset_controller.go

```diff
@@ -47,9 +67,50 @@ type ClusterSetReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
```

Comment:
you can remove default comments added by kubebuilder and add more description about the logic of clusterset reconciler.